msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -997,6 +997,52 @@ The `extensions.alternates` property is used to hide or prioritize specific in-m
 |`hide.customOfficeAddin`| | | | Configures how to hide an in-market add-in that isn't distributed through AppSource.|
 |`hide.customOfficeAddin.officeAddinId`|String | 64 characters | ✔️ | Specifies the ID of the in-market add-in to hide. The GUID is taken from the app manifest `id` property if the in-market add-in uses the JSON app manifest. The GUID is taken from the `<Id>` element if the in-market add-in uses the XML app manifest. |
 
+## actions
```

Comment:
We should disambiguate between actions feature and the OE actions under "extensions.runtimes" property. It can be confusing, I did a Ctrl+F and found the OE actions. I will leave it to Irene on when we should take this up. If we don't take this up now, I suggest we add this to the backlog and track this.